packages/design-system/src/components/TextField/TextField.js

```diff
@@ -58,6 +58,7 @@ const TextField = props => {
 
 TextField.propTypes = {
   labelText: PropTypes.string,
+  /**  specifies additional decorative element rendered at the end of the label */
```

Comment:
I know that it's duplicating code but could you add this comment also in NumericInputField.js, SelectField.js, MultiselectField.js, TextareaField.js? We will refactor it later.